Flutter/calendar/appointments.md

```diff
@@ -937,7 +937,7 @@ You can customize the displaying time format in the appointment widget in the mo
 You can get the list of visible appointments by using the [getVisibleAppointments](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/CalendarDataSource/getVisibleAppointments.html) method available in the Calendar data source.
 
 {% tabs %}
-{% highlight dart hl_lines="19" %}
+{% highlight dart hl_lines="19 20 21" %}
```

Comment:
https://help.syncfusion.com/flutter/calendar/appointments#get-pattern-appointment

Highlight is wrong

Reply:
Highlighted properly.